eZ/Publish/Core/Repository/Helper/DomainMapper.php

```diff
@@ -189,6 +189,7 @@ public function buildDomainFields(array $spiFields, $contentType, array $languag
                         ->fromPersistenceValue($spiField->value),
                     'languageCode' => $spiField->languageCode,
                     'fieldDefIdentifier' => $fieldIdentifierMap[$spiField->fieldDefinitionId],
+                    'type' => $this->fieldTypeRegistry->getFieldType($spiField->type)->getFieldTypeIdentifier(),
```

Comment:
hmm, doesn't $spiField->type already contain the identifier here? alternatively field type is retrieved 3 lines above here already

Reply:
Uh, silly me :laughing: 
Thanks, amended via 9c0abfd. 

Also found more places for integration tests to check this new property, amended via b8c0f1c